troubadix/runner.py

```diff
@@ -99,6 +120,32 @@ def _report_bold_info(self, message: str):
     def _report_ok(self, message: str):
         self._term.ok(message)
 
+    def _process_plugin_results(self, results: OrderedDict):
+
```

Comment:
IMHO it is important to specify the internal types of the Dict. And also the method doesn't care about the OrderedDict "interface" here and just expects a dict. Btw. dicts are ordered too nowadays.
```suggestion
    def _process_plugin_results(self, results: Dict[str, List[LinterMessage]]) -> None:
```

Reply:
You have introduced the `OrderedDict` in `FileResults` ;)